.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -4,25 +4,26 @@ about: Create a report to help us improve
 title: ''
 labels: bug
 assignees: ''
-
 ---
 
 **Describe the bug**
 A clear and concise description of what the bug is.
 what is the current behavior
 
 **What is the expected behavior?**
- What is the motivation / use case for changing the behavior?
+What is the motivation / use case for changing the behavior?
 
 **To Reproduce**
 Steps to reproduce the behavior:
 Please provide a minimal working example of the bug if possible.
 
 **Environment (please complete the following information):**
- - Operating System: [all | Windows | Mac | Linux]
- - Language Extension: [all | Python | MATLAB | Java | Octave| C | C++]
- - what compiler or setup process did you use
- - HELICS version: <!-- 1.3.0 (07-31-18) -->
+
+- Operating System: [all | Windows | Mac | Linux]
+- Language Extension: [all | Python | MATLAB | Java | Octave| C | C++]
+- what compiler or setup process did you use
+- HELICS version: <!-- 1.3.0 (07-31-18) -->
+
```

Comment:
Do you know why it adds a new line on line 21 but not on line 15? I am guessing because of the paragraph versus a bulleted list. It would be nice to consistently add a new line around paragraphs after headings.

Reply:
Looking at the other changes in the PR, I think it is because this heading style using ```**``` instead of ```##```. So in this case, the ```**``` is just a bold object.

Reply:
Do we want this to be a bold object or a heading.  There are a number of other fixes to make that are not automatic so I will likely be editing some of the files.